lms/services/h_api.py

```diff
@@ -61,18 +59,13 @@ def get_user(self, username):
 
         return HUser(username=username, display_name=user_info["display_name"])
 
-    # pylint: disable=too-many-arguments
-    # (data will be removed soon)
-    def _api_request(self, method, path, data=None, headers=None, body=None):
+    def _api_request(self, method, path, body=None, headers=None):
         """
         Send any kind of HTTP request to the h API and return the response.
 
-        :param method: the HTTP request method to use, for example "GET",
-                       "POST", "PUT", "PATCH" or "DELETE"
+        :param method: the HTTP request method to use, (e.g. "GET")
         :param path: the h API path to post to, relative to
-                     ``settings["h_api_url_private"]``, for example:
-                     ``"users"`` or ``"groups/<GROUPID>/members/<USERID>"``
-        :param data: the data to post as JSON in the request body
+                     `settings["h_api_url_private"]` (e.g. "users")
```

Comment:
Slimmed down the examples a bit here. It's odd to add examples we don't actually have in the code anymore